deploy/kubernetes/operator/api/uniffle/v1alpha1/remoteshuffleservice_types.go

```diff
@@ -359,7 +359,7 @@ type RemoteShuffleService struct {
 
 //+kubebuilder:object:root=true
 
-// RemoteShuffleServiceList contains a list of RemoteShuffleService
+// RemoteShuffleServiceList contains a list of  RemoteShuffleService
```

Comment:
Please revert unrelated changes.

```suggestion
// RemoteShuffleServiceList contains a list of RemoteShuffleService
```

Reply:
This change will trigger k8s ci. So we shouldn't revert before we verify the correctness of k8s integration pipeline.